plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgresTaskFailureRecoveryTest.java

```diff
@@ -22,4 +25,38 @@ public TestPostgresTaskFailureRecoveryTest()
     {
         super(RetryPolicy.TASK);
     }
+
+    @Test
+    @Override
+    protected void testDeleteWithSubquery()
+    {
+        // TODO: support merge with fte
```

Comment:
Is there issue on github about solving this TODO? How to reproduce the problem?

Reply:
There is https://github.com/trinodb/trino/issues/23345 
It seems can not reproduce, the current TASK level fte for MERGE won't fail

Reply:
So maybe it is now supported?

Reply:
Yes, seems, but it out of my understanding. I need to read what does the TASK level fte actual do. For now, from the test it says it is supported.